### DISCUSSION TITLE
Charts and Graphs in .NET MAUI

### DISCUSSION THREAD

**Discussion by nogginbox** [dotnet/maui#4468] [Q&A]
I'd like to add some charts to my app. Are there any open source chart drawing libraries out there (possibly ones that were developed for Xamarin Forms) that I can use to add some charts in .NET MAUI?

I'm not expecting them to be production ready yet as these are early days, but is there anything out there being worked on at the moment that is looking promising?

**Comment by Lorymi** [comment]
https://github.com/beto-rodriguez/LiveCharts2

**Reply by nogginbox** [reply]
These look very promising. Will try them out.

**Reply by canperk** [reply]
currently it is not working with IOS and issues are not answered

**Reply by nogginbox** [reply]
My MAU App is for Windows and Mac, so I've not tried these on iOS. At the time of my last comment, they weren't working, but I tried them again a few months and releases later and they were. So I'd hold out hope that they do fix the issues you are seeing.

**Reply by Alex90VP** [reply]
> в настоящее время он не работает с IOS, и вопросы не решаются

подтверждаю, а так хотелось бы

**Reply by beto-rodriguez** [reply]
> currently it is not working with IOS and issues are not answered

I am sure it at least used to work on iOS, I have a test app installed on an ipad, I will check the issue you have soon.

**Reply by canperk** [reply]
> > currently it is not working with IOS and issues are not answered
> 
> 
> 
> I am sure it at least used to work on iOS, I have a test app installed on an ipad, I will check the issue you have soon.

https://github.com/beto-rodriguez/LiveCharts2/issues/800

Here is the issue... fyi

**Reply by beto-rodriguez** [reply]
Thanks, I will check that out, notice it seems that works on IOS according to this issue https://github.com/mono/SkiaSharp/issues/2291.

There is also a chance that the issue in on MAUI or SkiaSharp, probably trying another version of any of those will fix the issue in your case.

**Reply by canperk** [reply]
moved to essential issue at [here](https://github.com/beto-rodriguez/LiveCharts2/issues/800)

**Comment by codeaphex** [comment]
See https://github.com/CommunityToolkit/Maui/discussions/232

**Comment by jsuarezruiz** [comment]
There are some chart controls in AlohaKit Controls https://github.com/jsuarezruiz/AlohaKit.Controls

**Reply by Alex90VP** [reply]
net maui 6.0 windows -> local iPhone 11 is working! Thanks

**Comment by harvey-hang** [comment]
any update ?

**Comment by ratocinoue** [comment]
I found it introduced on the .NET official blog.
https://devblogs.microsoft.com/dotnet/dotnet-maui-welcomes-syncfusion-open-source-contributions/
Syncfusion's MAUI library, which includes controls such as Charts, is now free and open source.
https://github.com/syncfusion/maui-toolkit
